samples/vxflex_v230_k8s_121.yaml

```diff
@@ -108,7 +108,7 @@ spec:
       #   effect: "NoSchedule"
 
     initContainers:
-      - image: dellemc/sdc:3.6
+      - image: dellemc/sdc:3.6.0.6
```

Comment:
Why are we updating the samples of config version 2.3? Same comment applies for all samples related to 2.3

Reply:
The change does not have any relation with driver version and also the orchestrator. it is basically related to underlying os. previously the sdc:3.6 supports the Automatic sdc installation on RHCOS now it is supported on RHEL8.4,RHEL 7.9. So i am updating all the config version files.